node-template/runtime/src/lib.rs

```diff
@@ -33,7 +33,7 @@ pub use sr_primitives::BuildStorage;
 pub use timestamp::Call as TimestampCall;
 pub use balances::Call as BalancesCall;
 pub use sr_primitives::{Permill, Perbill};
-pub use support::{StorageValue, construct_runtime, parameter_types};
+pub use support::{StorageValue, construct_runtime, parameter_types, traits::Randomness};
```

Comment:
Why do we need the trait here?

Reply:
Otherwise have this error
```
error[E0599]: no function or associated item named `random_seed` found for type `srml_randomness_collective_flip::Module<Runtime>` in the current scope
   --> node/runtime/src/lib.rs:594:30
    |
594 |             RandomnessCollectiveFlip::random_seed()
    |                                       ^^^^^^^^^^^ function or associated item not found in `srml_randomness_collective_flip::Module<Runtime>`
    |
    = help: items from traits can only be used if the trait is in scope
help: the following trait is implemented but not in scope, perhaps add a `use` for it:
```

Reply:
Ahh it is used there. Okay :)